park_api/cities/Rosenheim.py

```diff
@@ -0,0 +1,58 @@
+from bs4 import BeautifulSoup
+# from park_api.util import convert_date
+from park_api.geodata import GeoData
+from park_api.util import utc_now
+
+# This loads the geodata for this city if <city>.geojson exists in the same directory as this file.
+# No need to remove this if there's no geodata (yet), everything will still work.
+geodata = GeoData(__file__)
+
+# This function is called by the scraper and given the data of the page specified as source in geojson above.
+# It's supposed to return a dictionary containing everything the current spec expects. Tests will fail if it doesn't ;)
+def parse_html(html):
+
+    # BeautifulSoup is a great and easy way to parse the html and find the bits and pieces we're looking for.
+    soup = BeautifulSoup(html, "html.parser")
+
+    data = {
+        "last_updated": utc_now(),     # not found on site, so we use something else
+        # URL for the page where the scraper can gather the data
+        "lots": []
+    }
+
+    # load the JSON-file:
+    urlHD = 'https://www.rosenheim.de/index.php?eID=jwParkingGetParkings'
+    headerHD={'Accept': 'application/json; charset=utf-8', 
+               'User-Agent': 'ParkAPI v%s - Info: %s' %(env.SERVER_VERSION, env.SOURCE_REPOSITORY) }
+    req = urllib.request.Request(url=urlHD, headers=headerHD)
+    webURL = urllib.request.urlopen(req)
+    data=webURL.read()
+    dataJSON=json.loads(data)
```

Comment:
Why do you call urllib manually instead of using the argument `html` that provides exactly the same data?

Reply:
Because of the headers, which can't be specified otherwise.

Reply:
sorry for beeing so late with a reply.

I don't know how to read this via the usual way. I have to specify the type JSON to get the data.

PS: I don't like this approach either.

Reply:
what can I do to fix this error:
`TypeError: the JSON object must be str, not 'bytes'`

here everything works fine.

Reply:
OK, seems to work now.

Reply:
@kiliankoe I see. I thought those were only the headers we're already sending but that's not the case for the `Accept` header.